### PR TITLE
Fetch Movies

### PR DESCRIPTION
**Pull request by Juliana-Dimeglio**
Movies fetch from https://www.themoviedb.org/ api
Captura de pantalla 2021-06-24 105938
Captura de pantalla 2021-06-24 110013

### CODE REVIEW

File: lib/src/bloc/movies_bloc.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+
+import '../model/movie.dart';
+
+import '../resources/movie_repository.dart';
+
+class MoviesBloc {
+  final _repository = MovieRepository();
```
Comment (MarcosCGlobant) [inline lib/src/bloc/movies_bloc.dart]: you should rename "repository" to make it more representative

File: lib/src/model/movie.dart [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    print(parsedJson['results'].length);
+    _page = parsedJson['page'];
+    _totalResults = parsedJson['total_results'];
+    _totalPages = parsedJson['total_pages'];
+    List<Result> temp = [];
+    for (int i = 0; i < parsedJson['results'].length; i++) {
+      Result result = Result(parsedJson['results'][i]);
+      temp.add(result);
+    }
+    _results = temp;
```
Comment (MarcosCGlobant) [inline lib/src/model/movie.dart]: you should avoid shortening words, in "temp" you can use a more representative word like "temporalList" or something like that

File: lib/src/resources/movie_api_provider.dart [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import '../model/movie.dart';
+
+class MovieApiProvider {
+  Client client = Client();
+  final _apiKey = "1124adc80a8b854f1da261e38d081ce3";
+  final _baseUrl = "http://api.themoviedb.org/3/movie";
+
+  Future<Movie> fetchMovieList() async {
+    final response = await client.get(Uri.parse(
+        _baseUrl + "/popular?api_key=" + _apiKey + "&language=en-US&page=1"));
```
Comment (MarcosCGlobant) [inline lib/src/resources/movie_api_provider.dart]: you can add all this strings into a strings constants file, to avoid hardcodes

**Review by FranciscoGaraventa — APPROVED** [review]
After resolving Harry comments, LGTM

File: lib/src/resources/movie_api_provider.dart (hunk 3 above)
Comment (Juliana-Dimeglio) [reply lib/src/resources/movie_api_provider.dart]: Fixed

File: lib/src/model/movie.dart (hunk 2 above)
Comment (Juliana-Dimeglio) [reply lib/src/model/movie.dart]: Fixed

File: lib/src/bloc/movies_bloc.dart (hunk 1 above)
Comment (Juliana-Dimeglio) [reply lib/src/bloc/movies_bloc.dart]: Fixed

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**